src/Chapter1.hs

```diff
@@ -519,9 +522,9 @@ branches because it is an expression and it must always return some value.
 👩‍🔬 Due to lazy evaluation in Haskell, only the expression from the branch
   satisfying the check will be returned and, therefore, evaluated.
 -}
-closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
 
+closestToZero :: Int -> Int -> Int
+closestToZero x y = if (abs x) < (abs y) then x else y
```

Comment:
Not having to use brackets here will take some time getting used to.
Right now, the line with the brackets seems more readable to me because of how I'm used to writing code. 
I will keep in mind that it's cleaner this way for Haskell and pay attention to this when working on the next chapters. :)

Reply:
Haha, I see now, this is what functional programming did to me 😄 I will keep that in mind as well! This is indeed just a different way to "think in the language". Also, these are just suggestions to let you know about how it is possible to do in Haskell. Of course, you can write it with parenthesis as well, whichever you feel more comfortable with at the moment!